platform/solr/solr-query/pom.xml

```diff
@@ -45,17 +45,17 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.0</minimum>
+                                            <minimum>0.73</minimum>
```

Comment:
Sorry. I wasn't clear. I meant between the different _types_ of coverage. You've got 25% for branch all the way up to 73% for instructions. It's possible to be that big a range, but it seems weird. If that's what it is, 👍